DevProject/ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json

```diff
@@ -2,6 +2,22 @@
     "m_Name": "Settings",
     "m_Path": "ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json",
     "m_Dictionary": {
-        "m_DictionaryValues": []
+        "m_DictionaryValues": [
```

Comment:
Not sure what to do with this. It got added in https://github.com/Unity-Technologies/ml-agents/commit/7718dfe11da14b5c21dd5c32e4ce2a3a03ed8ca4#diff-41dd07c8a0392e1f6c2a7ba1277f3426cdc92aa8258e563912c39db66800ade0 but wasn't computing coverage on the assemblies we care about.

Reply:
yeah, I think it's just for local runs of code coverage